selenium_tests/course_info/course_info_base_test_case.py

```diff
@@ -10,11 +9,20 @@
     import AccountAdminDashboardPage
 from selenium_tests.course_info.page_objects.course_info_search_page_object \
     import CourseSearchPageObject
+from selenium_tests.course_info.page_objects.course_people_page_object \
+    import CoursePeoplePageObject
+
+
+# Common files used for all Manage People test cases
+TEST_USERS_WITH_ROLES = join(dirname(abspath(__file__)),
```

Comment:
From the variable name, I'd expect this to actually have users/roles in it.  Maybe rename to `TEST_USERS_WITH_ROLES_PATH`?